articles/mfa/guides/mfa-api/phone.md

```diff
@@ -63,17 +69,17 @@ If successful, you'll receive a response like the one below:
 }
 ```
 
-If you get a `User is already enrolled error`, the user already has an MFA factor enrolled. Before associating another factor with the user, you need to challenge the user with the existing factor.
+If you get a `User is already enrolled error`, is because the user already has an MFA factor enrolled. Before associating it another factor, you need challenge the user with the existing one.
 
 #### Recovery Codes
 
 <%= include('../../_includes/_recovery_codes') %>
 
-### 3. Confirm the SMS enrollment
+### 3. Confirm the SMS or Voice enrollment
 
-The user should receive an SMS containing the 6-digit code, which they can provide to the application.
+Users should receive an message with a 6-digit code that need to provide to the application.
```

Comment:
I would change `Users should receive an message with a 6-digit code that need to provide to the application.` to `Users should receive an message with a 6-digit code that they need to provide in the application.` (adding "they").